corehq/apps/reports/views.py

```diff
@@ -158,6 +159,12 @@
     group_filter,
     users_matching_filter,
 )
+from corehq.apps.style.decorators import (
+    use_bootstrap3,
+    use_jquery_ui,
+    use_jquery_ui_multiselect,
+    use_select2,
```

Comment:
ok. I think you can also just rebase https://github.com/dimagi/commcare-hq/pull/10397 on top of this after and prevent that. I don't like this approach as much because someone coming along later could easily delete them to clean up and then you could get into a broken state after a merge.